bundles/org.openhab.binding.deconz/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```

Comment:
Can you remove this files (see  #5224)?

Reply:
@J-N-K It is additional work for people to remove those files. I'd say we accept them as is (as long as the .gitignore is not in place anyway) and remove them altogether in one PR.